tests/e2e/tidbcluster/serial.go

```diff
@@ -846,7 +846,11 @@ var _ = ginkgo.Describe("[Serial]", func() {
 
 				ginkgo.By("Deploy original TiDB cluster")
 				utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc, 10*time.Minute, 5*time.Second)
-				selector := MustGetLabelSelectorForComponents(tcName, label.DiscoveryLabelVal, label.PumpLabelVal) // ingore discovery and pump
+				selector := MustGetLabelSelectorForComponents(tcName,
+					label.DiscoveryLabelVal,
+					label.PumpLabelVal,
+					label.TiCDCLabelVal, // ingore ticdc because of PR #4494
```

Comment:
Do we still need to add this as we have updated the code in `tests/pkg/fixture/fixture.go`?